helpers/ShareArticle.js

```diff
@@ -0,0 +1,73 @@
+import Debug from 'debug';
+import dotenv from 'dotenv';
+import Postals from './Postals';
+
+const debug = Debug('dev');
+dotenv.config();
+/**
+ *
+ *
+ * @export
+ * @class EmailVerification
+ */
+export default class ShareArticle {
+  /**
+   *
+   * @static
+   * @param {string} requestInfo - request information
+   * @param {string} recipient - recipient of the email
+   * @param {string} title - title of the article
+   * @param {string} slug - article slug
+   * @memberof ShareArticleByMail
+   * @returns {function} - returns a function call
+   */
+  static shareArticleByMail(requestInfo, recipient, title, slug) {
+    const { firstName, lastName, id } = requestInfo.user;
+    const url = `${requestInfo.protocol}://${requestInfo.get('host')}/api/v1/articles/${id}/${slug}`;
+    const message = `<p>  
+                    ${firstName} ${lastName} shared this article <b>${title}</b> from Author's Haven with you,
+                  </p>
+                  <p>
+                  Click <a href= ${url}> ${title}</a> to view
+                  </p>`;
+    Postals.sendEmail(
+      recipient,
+      'mazus.ah@gmail.com',
+      'Welcome to Author\' haven',
```

Comment:
Hey, this mail subject may be a bit confusing to the user

Reply:
> Hey, this mail subject may be a bit confusing to the user

I missed the 's', I would fix that. Thanks

Reply:
That's not what I meant, I mean that the subject of the email will be `Welcome to Author's Haven` when the message content is a shared article, wouldn't it make more sense if it was a subject more like `Author's Haven: Article Shared With You`?  or something that just basically ties with the content of the email

Reply:
Okay. I have fixed that already

Reply:
Great! You're a rockstar!!